static/refcache.json

```diff
@@ -5587,6 +6515,9 @@
     "StatusCode": 206,
     "LastSeen": "2023-06-29T18:39:49.775529-04:00"
   },
+  "https://rubygems.org/gems/opentelemetry-instrumentation-redis": {
+    "StatusCode": 200,
+    "LastSeen": "2024-01-08T12:17:25.018558+01:00"
   "https://rubygems.org/gems/opentelemetry-instrumentation-all": {
```

Comment:
These lines are misformatted.

Reply:
hopefully fixed in https://github.com/open-telemetry/opentelemetry.io/pull/3730/commits/dbef3153d066cff1514140411fb6639173d9699f